.vscode/settings.json

```diff
@@ -1,12 +1,16 @@
 {
   "cSpell.words": [
-    "NOINLINE",
-    "WHNF",
+    "dflipflop",
```

Comment:
Or in Dutch, _de flipflop_

Reply:
I think this thing is properly spelled "D flip-flop", not as a single word.

I don't have time to review further now.

Reply:
Looking at the rest of the words, I think it's spelled front-end, with a dash, like the flip-flop.

Reply:
We have a function called `dflipflop`, so it comes up often :sweat:

Reply:
You can't have VSCode ignore all words that are typeset in a fixed width font? Just an idea, I dunno. Seems pretty dumb that it is spell checking code.

Reply:
`dflipflop` is mentioned in the documentation.

Reply:
So are the names of other functions. I'm saying this should be spell checked:
> This is a variant of dflipflop

but this should be left alone:
> This is a variant of `dflipflop`

because the fixed width font indicates this, in all likeliness, is code.

But I don't use VSCode and I don't use spell checking, so I'm just saying it sounds dumb that it's trying to spell check something that by its formatting is very likely to not be prose.

Reply:
I appreciate the fact the most replies are on my dumb joke comment 😅

Reply:
:cry:

Reply:
In any case I'll delete it from the additions. People will have to suffer through the red scribbles.

Reply:
Maybe we should correct the spelling of frontend (should be: [front-end](https://www.oxfordlearnersdictionaries.com/definition/english/front-end_2) (adjective), [front end](https://www.oxfordlearnersdictionaries.com/definition/english/front-end_1) (noun)) and remove that misspelled word from the word list as well. In a separate PR. Unless someone can show a dictionary that spells it as such.

And both `NOINLINE` and `fmap` in the word list are of the same class as `dflipfop`: they denote code rather than prose, and if I would use them in documentation I'm writing, I would format them as code with a fixed-width font.

> We have a function called dflipflop, so it comes up often 😓

```
$ git grep dflipflop
clash-cores/src/Clash/Cores/LatticeSemi/ICE40/IO.hs:dflipflopE
clash-cores/src/Clash/Cores/LatticeSemi/ICE40/IO.hs:dflipflopE = delay (deepErrorX "dflipflopE: undefined initial value")
clash-cores/src/Clash/Cores/LatticeSemi/ICE40/IO.hs:    mux latch_dIn_0 (dflipflopE pkgPinInRead) pkgPinInRead
clash-prelude/src/Clash/Explicit/Reset.hs:-- For synchronous domains two sequential dflipflops are used:
clash-prelude/src/Clash/Explicit/Signal.hs:  , dflipflop
clash-prelude/src/Clash/Explicit/Signal.hs:dflipflop
clash-prelude/src/Clash/Explicit/Signal.hs:dflipflop = \clk i ->
clash-prelude/src/Clash/Explicit/Signal.hs:    (deepErrorX "First value of dflipflop undefined")
clash-prelude/src/Clash/Explicit/Signal.hs:{-# INLINE dflipflop #-}
clash-prelude/src/Clash/Signal.hs:  , dflipflop
clash-prelude/src/Clash/Signal.hs:dflipflop
clash-prelude/src/Clash/Signal.hs:dflipflop =
clash-prelude/src/Clash/Signal.hs:  E.dflipflop (fromLabel @(HiddenClockName dom))
clash-prelude/src/Clash/Signal.hs:{-# INLINE dflipflop #-}
```

I see one occurrence (`Reset.hs`) and that can trivially be replaced by _D flip-flop_. This PR adds one further occurrence as yet.

I'm personally not even requesting you remove them from this word list even though I do think they probably shouldn't be on there. It feels like a slippery slope: exactly how often does a word need to occur before we add it? At the most extreme, do we add all functions in `clash-prelude` as well as `base`? (Of course we don't)

Perhaps you can just mentally reconcile the red scribble with an indication that you need to format something as a fixed-width font? :-D